websites-css/project.css

```diff
@@ -2,20 +2,82 @@
     box-sizing: border-box;
 }
 
+body {
+    display: flex;
+    height: 100%;
+    flex-direction: column;
+}
+
+html {
+    height: 100vh;
+}
+
 .header {
     background-color: var(--color-header-bg);
+    margin: 0;
     padding: 16px;
+    
 }
 
-.header__page-title {
-    color: var(--color-header-text);
-}
 
-.main {
+.main-content {
     flex-grow: 1;
 }
 
-footer {
+.grid-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr); 
+    grid-template-rows: repeat(2, 280px) 200px;
+    gap: 10px; 
+    max-width: 800px; 
+    margin: 0 auto; 
     
-    padding: 16px;
-}
\ No newline at end of file
+}
+
+
+.box {
+    background-color: var(--color-content-section);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 1px solid #000;
+    font-size: 1.5rem;
```

Comment:
Switch to global variable for scaleability, or use already existing variable --font-size-h3, which also is 1.5rem.

Reply:
its change and update